CHANGELOG.md

```diff
@@ -66,6 +66,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## 4.2.0 (2018-07-18)
 
+* new
+  * added Authlogic::Session::Cookies.encrypt_cookie option (default: false)
```

Comment:
cc @jaredbeck to confirm, but I think this might be better under the `Added` section nowadays.

Reply:
good point, done.